modules/planning/tasks/traffic_decider/pull_over.cc

```diff
@@ -148,31 +172,31 @@ bool PullOver::CheckPullOver() {
   return true;
 }
 
-bool PullOver::BuildPullOverStop(const PointENU& stop_point,
+int PullOver::BuildPullOverStop(const PointENU& stop_point,
                                  double stop_heading) {
   // check
   const auto& reference_line = reference_line_info_->reference_line();
   common::SLPoint sl;
   reference_line.XYToSL(stop_point, &sl);
   if (sl.s() < 0 || sl.s() > reference_line.Length()) {
```

Comment:
Is it necessary to double check sl.s() here?

Reply:
it is always recommended to add checks. you can't assume the parameters passed in are all legit.